source/features/collapsible-content-button.tsx

```diff
@@ -44,6 +45,12 @@ function init(): void {
 	}
 }
 
+function init(): void {
+	delegate(document, '.rgh-collapsible-content-btn', 'click', addContentToDetails);
+	addButtons();
+	onCommentEdit(addButtons);
```

Comment:
As far as I know, GitHub duplicates a form that already exists on the page (this was the case for new reviews at least) so that means we only had to edit one form once and then GitHub would just clone it. 

Is that no longer the case? Or maybe here GitHub started loading the whole edit field when clicking Edit? If that's what's happening, then this is correct.

Reply:
> Is that no longer the case?

Doesn't seem like it:
 * the first comment of a conversation is created with a complete edit form on page load
 * all the other comments (regular & reviews) have an associated form with an `<include-fragment>` in it, which is loaded as soon as the user hovers on the "three dots" pop-up menu